content/posts/hiero_links_of_july_twentyith.md

```
@@ -0,0 +1,29 @@
+Below are links to some of the things that happened this past week. **Check it out!**
+
+---
+**[Angelina C](https://github.com/aceppaluni)** added [rejectToken](https://github.com/hiero-ledger/hiero-sdk-python/pull/2442) to the TCK module inside the Python SDK.
+
+**[Fantomasa](https://github.com/Fantomasa)** updated the Hiero [Solo Documentation](https://github.com/hiero-ledger/solo/pull/5049) to outline new variables introduced by Mirror Node database schemas. 
+
+**[Abhijeet](https://github.com/Abhijeet2409)** added a workflow to [check for broken links](https://github.com/hiero-ledger/hiero-website/pull/327) in the Hiero Website. 
+
+**[Dian Deskov](https://github.com/ddeskov-limechain)** fixed and reenabled [can set network name on custom network](https://github.com/hiero-ledger/hiero-sdk-js/pull/4267) test in the Hiero JavaScript SDK to correct flaky client integration testing. 
+
+**Want to be featured on the Hiero-Website?**
```

markdownlint-cli2 (0.23.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)